content/pricing/oss-project.md

```diff
@@ -0,0 +1,47 @@
+---
+title: OSS Project Pricing
+meta_desc: Pulumi's infrastructure as code SDK and SaaS products together provide plans for all organizations, including OSI licensed  open source projects.
+type: page
+layout: pricing-oss
+
+hero:
+    title: Pulumi Open Source Program
+    description: |
+        At the core of Pulumi is an open source project and an engaged community. Pulumi exists thanks
+        to the work of hundreds of open source contributors around the world. As a company, we are
+        deeply committed to being a good steward of the Pulumi open source project. As a way to
+        give back to the community, the Pulumi Open Source Program was created to help open source
+        projects and teams innovate and scale faster through cloud engineering.
+    cta_text: Request Access
+
+features:
+    title: Program Features
+    description: |
+        Projects can utilize the Pulumi open source version today for free if they prefer to manage
```

Comment:
I think this sentence should move to the end or maybe even be removed? we should lead with what they are getting as part of this program

Reply:
👍